apps/rotm/behaviours/recaptcha.js

```diff
@@ -0,0 +1,89 @@
+const { reCaptcha } = require('../../../config');
+const {RecaptchaEnterpriseServiceClient} = require('@google-cloud/recaptcha-enterprise');
+
+/**
+  * Create an assessment to analyze the risk of a UI action.
+  *
+  * projectID: Your Google Cloud Project ID.
+  * recaptchaSiteKey: The reCAPTCHA key associated with the site/app
+  * token: The generated token obtained from the client.
```

Comment:
Consider using JSDocs style
```suggestion
/**
 * Creates an assessment to analyze the risk of a UI action.
 *
 * @param {string} projectId - Google Cloud project ID.
 * @param {string} recaptchaSiteKey - reCAPTCHA site key associated with the site or app.
 * @param {string} token - Client-generated reCAPTCHA token.
 * @param {string} recaptchaAction - Action name associated with the token.
 */
```

Reply:
updated 👍